test/test_ref_compile.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import annotations
+
+import os
+import unittest
+
+import pytest
+import torch
+
+from . import test_examples
+import helion
+from helion._testing import TestCase
+
+
+class TestExamplesRefCompile(test_examples.TestExamples):
+    """Run all TestExamples tests in reference torch.compile mode via HELION_REF_COMPILE=1."""
+
+    # NOTE: All tests in TestExamples are run in ref torch.compile(fullgraph=True) mode by default in this test file.
```

Comment:
Currently all examples in TestExamples pass with ref eager mode and ref compile mode.